### DISCUSSION TITLE
When or why do I need to "enable jieba/pinyin filters in citekey patterns"?

### DISCUSSION THREAD

**Discussion by seaspeak** [retorquere/zotero-better-bibtex#2626] [Q&A]
As I know, ticking "Force citation key to plain text" equals Fold, which transliterate Chinese into Pinyin (looks like Pinyin), while "Enable jieba/pinyin" is not enabled. It looks like Fold does more than "jieba/pinyin" or transliterate.

So here comes the questions:

1. What effects do I miss if I "Force citation key to plain text" but disable "jieba/pinyin"?
2. What effects do I miss if I use filter `ideographs` or `fold` in Citation key formula, but diable "jieba/pinyin"?

**Comment by seaspeak** [comment]
If "jieba/pinyin" is not enabled, formula `pinyin` and `jieba` seems not working.

**Reply by retorquere** [reply]
That is correct.
